controllers/object_controls.go

```diff
@@ -1238,6 +1238,23 @@ func TransformToolkit(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpec, n
 
 	// configure runtime
 	runtime := n.runtime.String()
+	err = TransformDaemonsetForRuntime(obj, config, runtime)
```

Comment:
nit: what about renaming to `transformForRuntime()` to align with `transformforHostRoot` and `transformForDriverInstallDir` which I added in https://github.com/NVIDIA/gpu-operator/pull/747?

Reply:
Renamed